docs/user_guide/hopsworks/jupyter.rst

```diff
@@ -216,6 +216,92 @@ When you download a dataframe from spark to pandas with sparkmagic, it gives you
     Plot a local pandas dataframe using matplotlib and the magic %%local
 
 
+Jupyter notebooks with version control
+--------------------------------------
+
+Jupyter notebooks have become the lingua franca for data scientists. As with ordinary source code files, we should version them
+to be able to keep track of the changes we made or collaborate.
+
+**Hopsworks Enterprise Edition** comes with a feature to allow users version their notebooks with Git and interact with remote repositories such as
+repositories in GitHub. To authenticate in a remote service we use API keys which are safely stored in Hopsworks.
```

Comment:
@kouzant `Github ones. Authenticating against a remote service is done using API keys which...`